components/ActivityIndicator/examples/ActivityIndicatorExample.m

```diff
@@ -17,10 +17,10 @@
 #import <UIKit/UIKit.h>
 
 #import "ActivityIndicatorExampleSupplemental.h"
+#import "CatalogStyle.h"
```

Comment:
Here and below.
All of our examples are intended to be free of dependencies.  This allows clients to easily cut and paste the code into their own projects.  (This is also why we have the *Supplemental.h structure)
Do we want to change this strategy?